contracts/non-fungible-tokens/andromeda-crowdfund/src/contract.rs

```diff
@@ -87,6 +92,16 @@ pub fn reply(_deps: DepsMut, _env: Env, msg: Reply) -> Result<Response, Contract
 
 #[cfg_attr(not(feature = "library"), entry_point)]
 pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
+    let prev = get_contract_version(deps.storage)?;
+    let prev: Version = prev.version.parse().map_err(from_semver)?;
+    let v1: Version = "1.1.0".parse().unwrap();
```

Comment:
You should be able to get this from the [`CONTRACT_VERSION`](https://github.com/andromedaprotocol/andromeda-core/blob/efb2ea8cbc3334765a253f1768f61ef3d2d1c483/contracts/non-fungible-tokens/andromeda-crowdfund/src/contract.rs#L41) constant at the top of this file.